lerna-package-lock.json

```diff
@@ -2264,414 +2264,38 @@
-						"sockjs": "^0.3.21",
-						"sockjs-client": "^1.5.0",
-						"spdy": "^4.0.2",
-						"strip-ansi": "^3.0.1",
-						"supports-color": "^6.1.0",
-						"url": "^0.11.0",
-						"webpack-dev-middleware": "^3.7.2",
-						"webpack-log": "^2.0.0",
-						"ws": "^6.2.1",
-						"yargs": "^13.3.2"
-					},
-					"dependencies": {
-						"ws": {
-							"version": "6.2.2",
-							"resolved": "https://registry.npmjs.org/ws/-/ws-6.2.2.tgz",
-							"integrity": "sha512-zmhltoSR8u1cnDsD43TX59mzoMZsLKqUweyYBAIvTngR3shc0W6aOZylZmq/7hqyVxPdi+5Ud2QInblgyE72fw==",
-							"requires": {
-								"async-limiter": "~1.0.0"
-							}
-						}
-					}
-				},
-				"wrap-ansi": {
-					"version": "5.1.0",
-					"resolved": "https://registry.npmjs.org/wrap-ansi/-/wrap-ansi-5.1.0.tgz",
-					"integrity": "sha512-QC1/iN/2/RPVJ5jYK8BGttj5z83LmSKmvbvrXPNCLZSEb32KKVDJDl/MOt2N01qU2H/FkzEa9PKto1BqDjtd7Q==",
-					"requires": {
-						"ansi-styles": "^3.2.0",
-						"string-width": "^3.0.0",
-						"strip-ansi": "^5.0.0"
-					},
-					"dependencies": {
-						"ansi-regex": {
-							"version": "4.1.1",
-							"resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-4.1.1.tgz",
-							"integrity": "sha512-ILlv4k/3f6vfQ4OoP2AGvirOktlQ98ZEL1k9FaQjxa3L1abBgbuTDAdPOpvbGncC0BTVQrl+OM8xZGK6tWXt7g=="
-						},
-						"strip-ansi": {
-							"version": "5.2.0",
-							"resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-5.2.0.tgz",
-							"integrity": "sha512-DuRs1gKbBqsMKIZlrffwlug8MHkcnpjs5VPmL1PAh+mA30U0DTotfDZ0d2UUsXpPmPmMMJ6W773MaA3J+lbiWA==",
-							"requires": {
-								"ansi-regex": "^4.1.0"
-							}
-						}
-					}
-				},
-				"yargs": {
-					"version": "13.3.2",
-					"resolved": "https://registry.npmjs.org/yargs/-/yargs-13.3.2.tgz",
-					"integrity": "sha512-AX3Zw5iPruN5ie6xGRIDgqkT+ZhnRlZMLMHAs8tg7nRruy2Nb+i5o9bwghAogtM08q1dpr2LVoS8KSTMYpWXUw==",
-					"requires": {
-						"cliui": "^5.0.0",
-						"find-up": "^3.0.0",
-						"get-caller-file": "^2.0.1",
-						"require-directory": "^2.1.1",
-						"require-main-filename": "^2.0.0",
-						"set-blocking": "^2.0.0",
-						"string-width": "^3.0.0",
-						"which-module": "^2.0.0",
-						"y18n": "^4.0.0",
-						"yargs-parser": "^13.1.2"
-					}
-				},
-				"yargs-parser": {
-					"version": "13.1.2",
-					"resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-13.1.2.tgz",
-					"integrity": "sha512-3lbsNRf/j+A4QuSZfDRA7HRSfWrzO0YjqTJd5kjAq37Zep1CEgaYmrH9Q3GwPiB9cHyd1Y1UwggGhJGoxipbzg==",
-					"requires": {
-						"camelcase": "^5.0.0",
-						"decamelize": "^1.2.0"
-					}
 				}
 			}
 		},
 		"@fluidframework/agent-scheduler-previous": {
-			"version": "0.59.1000",
-			"resolved": "https://registry.npmjs.org/@fluidframework/agent-scheduler/-/agent-scheduler-0.59.1000.tgz",
-			"integrity": "sha512-2Vvr2Ofl+1yfPLal/kkwQyfatx2Q7sP+VbwvUIi3sAe8M86+e6Enk+cv8hddtk3GhtmycqhKvDcOeepXc8JB3w==",
+			"version": "npm:@fluidframework/agent-scheduler@0.59.2001",
```

Comment:
@anthony-murphy @curtisman @tylerbutler @CraigMacomber  - Anyone know what's up with this version format?  Is it ok to merge?  When I reinstalled on `next` before starting this change was made (see first commit in the branch)

Reply:
the format of this file isn't something i have great clarity on, so i don't know. but it seems to be passing status checks, so it builds, so it's probably fine

Reply:
If running npm i for other devs removes it, that will be a pain. I'll checkout this branch and see if that happens.

Reply:
I see no delta after running npm i locally, so I guess its working fine. No idea why it did this though.

Reply:
Thanks for checking that Craig!